kentico-michaelb/kontent-sample-app-conference-net
# Removes unused Model and fixes #7 and #13

### Motivation

fixes #7  and #13 

### Checklist


### How to test
Repro steps for Issue #7 

1. Go to the the TechConference project in Kentico Kontent
2. Remove the Asset from the Photo element on a Speaker Content Item
3. Launch the MVC application
4. Visit "Speakers" section (Example: http://~/Denver/Speakers)

Repro steps for Issue #13 

1. Go to '~/Denver/Agenda/why-should-i-structure-my-content'
2. Confirm that YouTube video loads without error
